supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -2,8 +2,30 @@
 [[specific-doc-types]]
 = Specific documentation types
 
+[[prerequisites]]
+== Prerequisites
+In contradistinction to normative technical writing guidelines, write prerequisites using the passive voice.
+
+*Reasoning:* The active voice turns a prerequisite into a procedural step.
```

Comment:
I don't think this reasoning is quite on the money. It depends on whether the action is significant or trivial imo. 

A significant action is one that would prevent the user from completing the procedure within a reasonable period of time. With the example in this PR, logging in with admin permissions is not significant. However it might take several hours or days to gain admin permissions, depending on the system.

"Ensure you have root permissions." or "Request root permissions for the system, if necessary." I feel like these are pre-reqs because, until that is complete, you cannot reasonably complete the procedure.

Logging in is a trivial task that could be the first step in a procedure. 

As I mentioned in my comment on the issue (https://github.com/redhat-documentation/supplementary-style-guide/issues/67#issuecomment-789575732), I think the guidance here is that we should help writers distinguish between what a pre-req is and what it is not.

I also think, at all times, technical documentation should be authoritative and concise. It seems to me this guidance advocates bad writing practices.